doc/env_vars.md

```diff
@@ -95,8 +95,8 @@ The following options are used for advanced database performance tuning and may
 
 | Env Var                         | Config                  | Description                                                                                                                                                                                                       | Default                                                                               |
 | ------------------------------- | ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------- |
-| `LRSQL_ADMIN_USER_DEFAULT`      | `adminUserDefault`      | The username of the account that seeds the account table, ie. added to the table upon initialization. Optional but **should be set in order to create other accounts**.                                           | Not set                                                                               |
-| `LRSQL_ADMIN_PASS_DEFAULT`      | `adminPassDefault`      | The password of the account that seeds the account table. Optional but **should be set in order to create other accounts**.                                                                                       | Not set                                                                               |
+| `LRSQL_ADMIN_USER_DEFAULT`      | `adminUserDefault`      | The username of the account that seeds the account table, ie. added to the table upon initialization. Optional but **should be set in order to create other accounts**. **NOTE: Not subject to validation.**      | Not set                                                                               |
+| `LRSQL_ADMIN_PASS_DEFAULT`      | `adminPassDefault`      | The password of the account that seeds the account table. Optional but **should be set in order to create other accounts**. **NOTE: Not subject to validation. Please choose a secure password.**                 | Not set                                                                               |
```

Comment:
Same as with the previous comment; I would change "Not subject to validation" to "Not subject to password length and character requirements," plus remove the redundant `**`.

Reply:
I did think that the sentence "Please choose a secure password" was redundant, but after re-reading the description it's actually fine. I just thought we already mentioned the user should create a secure password, but apparently we don't.

Reply:
3102d746bb0f2c7bd794c7b9c5a38e33b693e9f4